tests/modules/test_rnn.py

```diff
@@ -400,6 +400,10 @@ class TestNumberOfNodes:
     logging.basicConfig(level=logging.INFO, stream=sys.stdout)
 
     def test_number_of_calling_fq_for_lstm(self):
+        pytest.skip("Somehow certain quantizers in LSTM end up being "
```

Comment:
😞

Reply:
@ljaljushkin I've put some more effort into this and prohibited quantizer propagation for iteration scope nodes - should produce more consistent quantizer setups for RNNs, although probably somewhat less efficient.